base_ebill_payment_contract/views/ebill_payment_contract.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 Camptocamp SA
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+  <record model="ir.ui.view" id="ebill_payment_contract_form_view">
+    <field name="name">ebill.payment.contract.form</field>
+    <field name="model">ebill.payment.contract</field>
+    <field name="arch" type="xml">
+      <form>
+        <header>
+        </header>
+        <sheet>
+          <group>
+            <field name="transmit_method_id"/>
+            <field name="state"/>
+            <field name="date_start"/>
+            <field name="date_end"/>
+            <field name="partner_id"/>
+          </group>
+        </sheet>
+      </form>
+    </field>
+  </record>
+
+  <record model="ir.ui.view" id="ebill_payment_contract_search_view">
+    <field name="name">ebill.payment.contract.search</field>
+    <field name="model">ebill.payment.contract</field>
+    <field name="arch" type="xml">
+      <search>
+      </search>
```

Comment:
Same?

Reply:
I just added some search fields and filter :+1: